modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionMetadata.java

```diff
@@ -51,6 +51,8 @@ public BaseModel<?> deleteBaseModel(BaseModel<?> baseModel)
 
 	public String getRESTContextPath();
```

Comment:
@ccorreagg do you think it would make sense to merge all these methods in something like:

```java
JaxRsApplicationDescriptor getJaxRSAplicationDescriptor()
```

where the `JaxRsApplicationDescriptor` contains everythinng neded (name, version, path...).

Reply:
I also like your suggestion